pyuvsim/tests/test_mpi_uvsim.py

```diff
@@ -88,12 +88,15 @@ def test_run_paramfile_uvsim():
     os.remove(tempfilename)
 
     param_filename = os.path.join(SIM_DATA_PATH, 'test_config', 'param_1time_1src_testvot.yaml')
-    uvtest.checkWarnings(
-        pyuvsim.uvsim.run_uvsim, [param_filename],
-        nwarnings=11,
-        message=[SIM_DATA_PATH] * 10 + ['The default for the `center` keyword has changed'],
-        category=[astropy.io.votable.exceptions.W50] * 10 + [DeprecationWarning]
-    )
+    if six.PY2:
+        pyuvsim.uvsim.run_uvsim(param_filename)
+    else:
+        uvtest.checkWarnings(
+            pyuvsim.uvsim.run_uvsim, [param_filename],
+            nwarnings=1,
+            message=['The default for the `center` keyword has changed'],
+            category=[DeprecationWarning]
+        )
```

Comment:
I'd suggest using pytest to filter out this warning (put this right above the test: `@pytest.mark.filterwarnings("ignore:The default for the `center` keyword has changed")`) unless you really want to check that it is raised. It's going away soon in pyuvdata (the PR is close to merging) and doing it this way also introduces more six dependence which we're going to remove shortly.